fpga/src/main/scala/arty/Configs.scala

```diff
@@ -38,4 +38,10 @@ class WithArtyTweaks extends Config(
 class TinyRocketArtyConfig extends Config(
   new WithArtyTweaks ++
   new chipyard.TinyRocketConfig)
+
+class TinyRocketArtySimConfig extends Config( // any additional IO needed for VCS sim go here.
+  new TinyRocketArtyConfig ++
+  new chipyard.harness.WithTiedOffDebug ++
```

Comment:
I believe the ordering of these fragments should be reversed (TinyRocketArtyConfig last).

Reply:
Thanks, fixed this in commit.